qdrant-landing/content/documentation/private-cloud/configuration.md

```diff
@@ -110,9 +110,6 @@ operator:
       # Maximum burst for throttle.
       # Default is 500.
       burst: 500
-      # If set to true, the operator reconciles all watched CRs,
-      # fully ignoring operator.qdrant.com/version annotation on them
-      disableOperatorVersionCheck: true
```

Comment:
are you sure you want to drop this for private cloud?

Reply:
I don't want to drop the setting, but remove it from the docs. Private cloud customers should never change this value from the default.

Reply:
Agreed, sorry!